### PR TITLE
Fixes #3258 long author names breaks themes layout

### LINKED ISSUE #3258: Long author name breaks layout for Themes
Steps to reproduce:
1. Change your display name in a long one in AMO-stage https://addons.allizom.org/en-US/firefox/
2. Go to "My Themes" page and hover over any submitted theme

Expected results:
There are no display or layout issues.

Actual results:
The long author name brakes theme layout.

Notes/Issues:
Verified on FF48(Win 7). Issue is reproducing on AMO-stage and dev.
Screenshot for this issue: 
-at hover

-without hover
without_hover

#2275

### PR DESCRIPTION
**Pull request by Prashant-Surya**
Before:
----------
before1
-----------------------------------------------------------------------------------------------------------------------------------------
before2
------
After:
--------
after1
-----------------------------------------------------------------------------------------------------------------------------------------
after2

### CODE REVIEW

File: static/css/restyle/restyle.less [hunk 1]
```diff
@@ -1602,6 +1602,13 @@ h3.author .transfer-ownership {
   h4 {
     clear: both;
     font-size: 12px;
+
+    .author-names{
+      text-overflow: ellipsis;
+      overflow:hidden;
```
Comment (EnTeQuAk) [inline static/css/restyle/restyle.less]: Can you adapt the style we're using all over the file and add a space between the colon and the first character, e.g `overflow: hidden`?

File: static/css/restyle/restyle.less [hunk 2]
```diff
@@ -1602,6 +1602,13 @@ h3.author .transfer-ownership {
...
+      white-space:nowrap;
```
Comment (EnTeQuAk) [inline static/css/restyle/restyle.less]: Same here

**Comment by EnTeQuAk** [comment]
This looks great, just one small cleanup and I think we can merge this. Thanks!

**Comment by Prashant-Surya** [comment]
Should I add the space after colon and update pull request?

**Comment by EnTeQuAk** [comment]
yes please, that would be great

**Comment by Prashant-Surya** [comment]
I updated the commit. Please check

**Comment by EnTeQuAk** [comment]
Perfect, thanks a lot for the fix and update!

**Comment by atsay** [comment]
Thanks for the bug fix, Surya! I have added your contribution to our [recognition page](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#September_2016).

**Pull request merged**